hexa/workspaces/models.py

```diff
@@ -377,3 +378,32 @@ def env_key(self):
 
     class Meta:
         unique_together = [["connection", "code"]]
+
+
+DEFAULT_WORKSPACE_DESCRIPTION = """
+# {workspace_name}
+
+Welcome to your new "{workspace_name}" workspace!
```

Comment:
```suggestion
Welcome to your new workspace "{workspace_name}"!
```

Isn't it better like this ? 

Welcome to your new workspace "PHEOC Analysis..."!